ScienceJournal/ToneGenerator/ConductorSoundType.swift

```diff
@@ -53,9 +53,13 @@ class ConductorSoundType: PitchedSoundType {
       return nil
     }
 
-    let index = Int(floor((value - valueThreshhold) / (currentMaximumValue - valueThreshhold) *
+    let index = Int(exactly: floor((value - valueThreshhold) / (currentMaximumValue - valueThreshhold) *
```

Comment:
This line is > 100 char

Reply:
Updated Google-side and pushed here https://github.com/google/science-journal-ios/blob/c1e6e29eb240f92379a7a4bb78d2d5f44daa53d7/ScienceJournal/ToneGenerator/ConductorSoundType.swift